views/includes/sign-out-user-banner.html

```diff
@@ -1,8 +1,21 @@
 {% if userEmail %}
+  
+  {% set basePath = 'sign-out?page=' %}
+
+  {% set signOutPath = basePath + templateName %}
+  
+  {% if journey === 'remove' %}
+    {% set url = url if url else OE_CONFIGS.UPDATE_AN_OVERSEAS_ENTITY_URL %}
```

Comment:
Some comments here would be helpful (maybe). Why would the 'url', 'signOutPreviousPagePrefix' and 'signOutExtraQueryParams' not be set?